include/vkgcDefs.h

```diff
@@ -267,8 +274,8 @@ struct PipelineOptions {
 
   ShadowDescriptorTableUsage shadowDescriptorTableUsage; ///< Controls shadow descriptor table.
   unsigned shadowDescriptorTablePtrHigh;                 ///< Sets high part of VA ptr for shadow descriptor table.
-  bool nullDescriptor;                                   ///< If set, support VK_EXT_robustness2 to give defined
-                                                         ///  behavior for null descriptor
+  ExtendedRobustness extendedRobustness;                 ///< If set, support VK_EXT_robustness2 to give defined
```

Comment:
///< If set, support VK_EXT_robustness2 to give defined behavior for null descriptor and out-of-range access
->
///<  If set, support VK_EXT_robustness2 features

Move the feature explanations to ExtendedRobustness field comments.

Reply:
Done.